static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -398,59 +443,92 @@ $padding-wide: 100px;
 }
 
 .ingredients-list {
-    li:not(.ingredients-title) {
+    margin-bottom: $gs-baseline;
+    li:not(.recipe__article--list-title) {
         font-family: $f-sans-serif-text;
         font-size: 14px;
         border-top: 1px solid $grey;
-        margin-bottom: 6px;
+        margin-bottom: $gs-baseline / 2;
     }
 }
 
-.ingredients-title {
-    border-top: 4px solid $purple;
-    font-family: $f-serif-headline;
+.recipe__article--structured-byline, .recipe__article--list-title {
+    @include fs-headline(3)
+    font-size: 18px;
+    line-height: 18px;
     font-weight: 900;
-    color: $purple;
-    margin-bottom: 12px;
+    color: $features-main-2;
+    margin-bottom: $gs-baseline * 2;
+}
+
+.recipe__article--structured-byline {
+    margin-bottom: $gs-baseline / 2;
+    line-height: 18px;
+    margin-top: $gs-baseline / 2;
+
+    @include mq($from: desktop) {
+        margin-top: 0;
+        margin-bottom: $gs-baseline;
+    }
+}
+
+.recipe__article--list-title {
+    .ingredients-list & {
+        &:before {
+            width: 100%;
+        }
+    }
+
+    &:before {
+        display: block;
+        content: '';
+        height: 4px;
+        background-color: $features-main-2;
+        margin-bottom: $gs-baseline / 2;
+        @include mq($from: desktop) {
+            width: 50%;
+        }
+    }
 }
 
 .method-list {
-    margin-bottom: 24px;
+    margin-bottom: $gs-baseline * 2;
 
-    @include mq($from: tablet) {
-        padding-right: 50px;
+    @include mq($from: desktop) {
+        padding-right: calc(#{$gs-gutter / 2} * 5);
     }
 
     @include mq($from: wide) {
-        padding-right: $padding-wide;
+        padding-right: $gs-gutter * 8;
     }
 
-    li {
+    li:not(.recipe__article--list-title) {
+        font-family: $f-sans-serif-text;
         border-top: 1px solid $grey;
-        margin-bottom: 24px;
+        margin-bottom: $gs-baseline * 2;
     }
     .number {
+        @include f-headline;
         font-size: 48px;
-        font-family: $f-serif-headline;
         font-weight: 200;
         float: left;
-        line-height: 0px;
-        margin-top: 26px;
-        margin-right: 12px;
-        color: $purple;
+        line-height: 0;
+        margin-top: $gs-baseline * 2.2;
+        margin-right: $gs-baseline;
+        color: $features-main-2;
     }
 }
 
 .ad-wrapper {
-    background-color: $grey-two;
-    margin-left: -20px;
-    margin-top: 20px;
-    padding-top: 6px;
-    padding-bottom: 6px;
+    background-color: #f6f6f6;
```

Comment:
@gustavpursche $neutral-6 is #ececec which is too dark as this is the background colour for the ad container it needs to be lighter. Would light($neutral-6, 10%) be acceptable?